src/index.js

```diff
@@ -1,7 +1,309 @@
+// import react & leaflet
 import React from 'react';
 import ReactDOM from 'react-dom';
+import leafletPip from "@mapbox/leaflet-pip";
+import L from 'leaflet';
+import Modal from 'react-modal';
+
+// import components
+import Map from './map';
+import GameButtons from './gamebuttons';
+import BorderData from './border';
+import CountyData from './vtCountyPolygons';
+import LocationInfo from './locationinfo';
+import CountyList from './countyList';
+
+Modal.setAppElement('#root');
```

Comment:
This line could use some commenting
.

Reply:
In addition, give an overview of what Modal. Since it is a new library that not everyone knows about, it will help others understand how and why this library is used.